pom.xml

```diff
@@ -1268,8 +1279,12 @@
                     <version>1.9.9</version>
                     <configuration>
                         <excludedClasses>
+                            <!-- AutoValue generated classes. -->
                             <excludedClass>*.AutoValue_*</excludedClass>
+                            <!-- Refaster rule collections. -->
+                            <excludedClass>*.refaster*.*Rules*</excludedClass>
```

Comment:
Arguably this should be moved to a separate PR :shrug:

Reply:
It's okay to leave in IMO.